website/static/js/filepage/revisions.js

```diff
@@ -132,7 +132,7 @@ var FileRevisionsTable = {
     view: function(ctrl) {
         return m('#revisionsPanel.panel.panel-default', [
                 m('.panel-heading.clearfix', m('h3.panel-title', 'Revisions')),
-                m('.panel-body', {style:{'padding-right': '0px','padding-left':'0px'}}, (function() {
```

Comment:
Why were the `px`'s removed here?

From the [mithril docs](https://lhorie.github.io/mithril/mithril.html#usage):

>  Mithril does not auto-append units like px or % to any values.